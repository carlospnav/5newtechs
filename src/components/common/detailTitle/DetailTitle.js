import styled from "styled-components"
import css from "theme/variables"

export default styled.h2`
  font-family: ${css.font.roboto};
  font-size: 30px;
  font-weight: ${css.fontWeight.regular};
  color: white;
  letter-spacing: 1.1px;
`