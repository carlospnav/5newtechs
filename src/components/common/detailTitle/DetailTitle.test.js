import { shallow } from "enzyme"
import DetailTitle from "./DetailTitle"
import "jest-styled-components"

describe("<DetailTitle>", () => {
  const wrapper = shallow(<DetailTitle />)
  it("Should render a DetailTitle properly.", () => {
    expect(wrapper).toMatchSnapshot()
  })
})