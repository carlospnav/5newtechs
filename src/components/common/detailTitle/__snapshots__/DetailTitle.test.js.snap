// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailTitle> Should render a DetailTitle properly. 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  font-size: 30px;
  font-weight: 300;
  color: white;
  -webkit-letter-spacing: 1.1px;
  -moz-letter-spacing: 1.1px;
  -ms-letter-spacing: 1.1px;
  letter-spacing: 1.1px;
}

<h2
  className="c0"
/>
`;
